labs/lab2/chapter2.md

```diff
@@ -156,7 +156,6 @@ More generally:
 * Use a specific tag for the source image. Image updates may break things.
 * Place rarely changing statements towards the top of the file. This allows the re-use of cached image layers when rebuilding.
 * Group statements into multi-line statements. This avoids layers that have files needed only for build.
-* Use `LABEL run` instruction to prescribe how the image is to be run.
```

Comment:
why are you striking this? because podman doesn't support it directly? I think it is still very useful info

Reply:
mainly because we took the `atomic run` section out of the lab??

Reply:
yeah... i still think it is valuable, personally

Reply:
k, will add it back